kapre/time_frequency.py

```diff
@@ -595,19 +467,19 @@ class FreqAwareConv(Conv2D):
     When using this layer as the first layer in a model,
     provide the keyword argument `input_shape`
     (tuple of integers, does not include the sample axis),
-    e.g. `input_shape=(128, 128, 3)` for 128x128 RGB pictures
+    e.g. `input_shape=(128, 300, 1)?` for  128x300 spectograms with 1 channel
```

Comment:
kapre follows `(time, freq, channel)` in case you're confused.

Reply:
thanks! fixed now